docs/diagnostics/PX1099.md

```diff
@@ -59,27 +59,27 @@ The main idea of the PX1099 analysis is that all API usages in the checked code
-However, the `System.Environment.NewLine` property is whitelisted and is not reported by the PX1099 diagnostic. The white list makes it possible to avoid frequently reported false positives and to configure analysis more granularly.
+The diagnostic also supports a list of allowed APIs that are not reported even if they are recognized as forbidden. For example, the analysis reports the `System.Environment` type and all its members as forbidden. 
+However, the `System.Environment.NewLine` property is allowed and is not reported by the PX1099 diagnostic. The list of allowed APIs makes it possible to avoid frequently reported false positives and to configure analysis more granularly.
 
 ## Diagnostic Settings
 
-The PX1099 diagnostic provides advanced configuration options that you can use to customize the diagnostic behavior. You can disable the PX1099 diagnostic or configure custom files with banned and whitelisted APIs.
+The PX1099 diagnostic provides advanced configuration options that you can use to customize the diagnostic behavior. You can disable the PX1099 diagnostic or configure custom files with banned and allowed APIs.
 You can find the PX1099 settings in the following Acuminator section of Visual Studio settings: `Tools -> Options -> Acuminator -> Banned API`.
 ![Banned API settings](images/BannedApiSettings.png)
 
 The following settings are available:
 
 - `Enable PX1099 diagnostic for banned API`: Enables or disables the PX1099 diagnostic. By default, the diagnostic is enabled.
 - `Banned API File`: Specifies the path to the file with banned APIs.
-- `White List API File`: Specifies the path to the file with whitelisted APIs.
+- `Allowed API File`: Specifies the path to the file with allowed APIs.
 
-### Banned and Whitelisted APIs
+### Banned and Allowed APIs
 
-Acuminator comes with default lists of banned and whitelisted APIs. These files are deployed by Acuminator on your machine in the `<My Documents Folder>\Acuminator\Acumatica Banned API` folder. 
-The `Banned API File` and `White List API File` settings are initially set to these files. You can modify these files to customize the list of banned and whitelisted APIs.
+Acuminator comes with default lists of banned and allowed APIs. These files are deployed by Acuminator on your machine in the `<My Documents Folder>\Acuminator\Acumatica Banned API` folder. 
+The `Banned API File` and `Allowed API File` settings are initially set to these files. You can modify these files to customize the list of banned and allowed APIs.
 
-Note, that clearing the `Banned API File` and `White List API File` settings in Visual Studio will not prevent the PX1099 diagnostic from running. The diagnostic will use the default lists of banned and whitelisted APIs that are 
+Note, that clearing the `Banned API File` and `Allowed API File` settings in Visual Studio will not prevent the PX1099 diagnostic from running. The diagnostic will use the default lists of banned and allowed APIs that are 
```

Comment:
To avoid misinterpretation regarding stacked nouns (banned file, allowed file), maybe we can say "APIs" instead of "API"?

Reply:
Done